src/mscorlib/shared/System.Private.CoreLib.Shared.sources

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetsWindows Condition="'$(TargetsWindows)' != 'true'">false</TargetsWindows>
+    <TargetsUnix Condition="'$(TargetsUnix)' != 'true'">false</TargetsUnix>
+    <TargetsOSX Condition="'$(TargetsOSX)' != 'true'">false</TargetsOSX>
+  </PropertyGroup>
+  <ItemGroup>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\Buffers\*.cs"/>
+    <MscorlibSources Include="$(MSBuildThisFileDirectory)\System\IO\Error.cs"/>
```

Comment:
Can we not call it mscorlibSources because of it is not mscorlib anymore? The regular `<Compile Include=... />` maybe best because of it is likely work the best with tooling.